scripts/fix_keen_preprint_keys.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import sys
+import django
+from django.db import transaction
+django.setup()
```

Comment:
Minor: You can use `website.app.setup_django` which will set `DJANGO_SETTINGS_MODULE` in addition to calling `django.setup()`, but this is fine.